packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -33,15 +33,17 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
   await checkRateLimitAndThrowError({
     identifier: `invitedBy:${ctx.user.id}`,
   });
+  const team = await getTeamOrThrow(input.teamId);
+
+  const isOrg = team.isOrganization;
```

Comment:
Switched this order around so we can get if it is an organization based on teamId 

@ThyMinimalDev can you confirm this is the correct logic here?

Reply:
makes sense, probably better to check if the team is an org from database rathe than relying on input parameter